app/scripts/models/auth_brokers/fx-desktop.js

```diff
@@ -183,6 +219,7 @@ define([
         'unwrapBKey',
         'keyFetchToken',
         'customizeSync',
+        'declinedSyncEngines',
```

Comment:
FWIW I believe this is the terminology that sync uses in internally, you sync everything by default and have to explicitly specify the ones you *dont* want.  I agree it's confusing when presented out of its natural habitat like this.